companies/morganstanley.json

```diff
@@ -0,0 +1,56 @@
+{
+    "slug": "morganstanley",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Morgan Stanley & Co International plc",
+    "address": "Legal & Compliance Division\nMorgan Stanley & Co International plc\n20 Bank Street, Canary Wharf\nLondon E14 4AD\nUnited Kingdom",
```

Comment:
Please don't add the company name or ATTN line to the address, as these are added automatically by our website ([see](https://github.com/datenanfragen/data#addresses)).

Reply:
My bad, I was sure I had stripped it out.